app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] [REQUIRED] Great implementation @mullenlucas , the signup works just fine, however, it is a requirement to enable `:confirmable` . This will allow the application to ask for email confirmation because right now after my signup, it automatically signed me in. Kindly implement this before asking for another review.

Reply:
Thank you for the observation!